src/pastelid/common.h

```diff
@@ -166,7 +182,8 @@ namespace ed_crypto {
             message = str_str.str();
         }
 
-        const char *what() const noexcept {
+        const char *what() const noexcept
```

Comment:
char*

Reply:
why char*? 
this is how it is declared: 
https://en.cppreference.com/w/cpp/error/exception/what
what is missing here actually is override at the end

Reply:
I mean just formatting
const char* what() const noexcept
as I see you are trying to place or replace "&" or "*" in the left side. and I'm follow the same in my coding.